scripts/release.sh

```diff
@@ -349,6 +349,7 @@ function parse_flags() {
       --dot-release) is_dot_release=1 ;;
       --auto-release) is_auto_release=1 ;;
       --from-latest-nightly) FROM_NIGHTLY_RELEASE=latest ;;
+      --use-ko-docker-repo) USE_KO_DOCKER_REPO=1 ;;
```

Comment:
nit: why not just `--docker-repo my-docker-repo.foo`?

Reply:
do you mean change the flag name from `use-ok-docker-repo` to `docker-repo` and pass in the repo name there? If so, I guess, but now there are two different ways to specify the location and that seems even more confusing than a flag that means "I really mean it".

Reply:
Makes sense, ignore my comment.